Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.yml

```diff
@@ -0,0 +1,1824 @@
+      description: |-
+        The page number to view. Each page contains page_size values. Must be used along with page_size.
+        Default is '1'.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: "Specify a search query as a Lucene query string.\nExample: query=\"first_seen: [2022-10-01T07:00:58.030Z  TO 2022-10-21T06:41:54.452Z]\" \n\nNote: For more details refer to Symantec EDR (On-Prem) API document https://apidocs.securitycloud.symantec.com/#."
+      isArray: false
+      name: query
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specify a filters option in lieu of “query”. These filters will improve query performance. 
+      isArray: false
+      name: search_object
+      predefined:
+      - domain
+      - device_uid
+      required: false
+      secret: false
+    - default: false
+      description: Specify a search value. Supports a comma-separated query with multiple search values.
+      isArray: false
+      name: search_value
+      required: false
+      secret: false
+    deprecated: false
+    description: List of endpoint and domain association.
+    execution: false
+    name: symantec-edr-endpoint-domain-association-list
+    outputs:
+    - contextPath: SymantecEDR.EndpointDomainAssociation.data_source_url
+      description: |-
+        The URL that was accessed. Example: "http://www.westfallave.com/insight/cloudcar.exe".
+      type: String
+    - contextPath: SymantecEDR.EndpointDomainAssociation.data_source_url_domain
+      description: |-
+        Domain name of the accessed URL. Example: "westfallave.com".
+      type: String
+    - contextPath: SymantecEDR.EndpointDomainAssociation.device_ip
+      description: |-
+        The IPv6 or IPv4 address of the endpoint when this association was last updated. Example: "127.0.0.1".
+      type: String
+    - contextPath: SymantecEDR.EndpointDomainAssociation.device_name
+      description: |-
+        The host name or, if unavailable, the IP address of the endpoint when this association was last updated. Example: "170915-000020".
+      type: String
+    - contextPath: SymantecEDR.EndpointDomainAssociation.device_uid
+      description: |-
+        Unique ID of the endpoint that accessed this URL. Example: "04cfc04b-5c7a-4aa8-b95b-79be23f768f4".
+      type: String
+    - contextPath: SymantecEDR.EndpointDomainAssociation.frst_seen
+      description: |-
+        The timestamp (in ISO 8601 format) that specifies the creation time of the event that resulted in the creation of this association. Example: "2018-01-30T04:13:10.669Z"
+      type: String
+    - contextPath: SymantecEDR.EndpointDomainAssociation.last_seen
+      description: |-
+        The timestamp (in ISO 8601 format) that specifies the creation time of the event that resulted in the creation of this association. Example: "2018-01-30T04:13:10.669Z"
```

Comment:
Why is this description the same as the one for first_seen?  It should be different,